src/status_im/mailserver/core.cljs

```diff
@@ -521,23 +518,24 @@
                   {:mailserver/remove-peer address}
                   (set-current-mailserver))))))
 
+(defn check-connection! []
+  (re-frame/dispatch [:mailserver/check-connection-timeout]))
+
 (fx/defn check-connection
   "connection-checks counter is used to prevent changing
    mailserver on flaky connections
    if there is more than one connection check pending
       decrement the connection check counter
    else
       change mailserver if mailserver is connected"
-  [{:keys [db] :as cofx}]
+  [{:keys [db now] :as cofx}]
   ;; check if logged into multiaccount
   (when (contains? db :multiaccount)
-    (let [connection-checks (dec (:mailserver/connection-checks db))]
-      (if (>= 0 connection-checks)
+    (let [last-connection-attempt (:mailserver/last-connection-attempt db)]
```

Comment:
is the docstring still relevant after these changes?

Reply:
Nope, I need to update it thanks